maat-court-data-api/src/main/java/gov/uk/courtdata/dao/oracle/HardshipReviewType.java

```diff
@@ -17,26 +17,25 @@ public class HardshipReviewType implements ORAData, ORADataFactory
 
   protected MutableStruct _struct;
 
-  protected static int[] _sqlType =  { 2,2002,2,12,91,12,12,2002,2,2,2003,2003,2002,93 };
+  protected static int[] _sqlType =  { 2,2002,2,12,91,12,12,2002,2,2,2003,2002,93 };
   protected static ORADataFactory[] _factory = new ORADataFactory[14];
   static
   {
     _factory[1] = NewWorkReasonType.getORADataFactory();
     _factory[7] = HrSolicitorCostType.getORADataFactory();
     _factory[10] = HRSectionTabType.getORADataFactory();
-    _factory[11] = HRProgressTabType.getORADataFactory();
-    _factory[12] = AssStatusType.getORADataFactory();
+    _factory[11] = AssStatusType.getORADataFactory();
   }
   protected static final HardshipReviewType _HardshipReviewTypeFactory = new HardshipReviewType();
 
   public static ORADataFactory getORADataFactory()
   { return _HardshipReviewTypeFactory; }
   /* constructors */
   protected void _init_struct(boolean init)
-  { if (init) _struct = new MutableStruct(new Object[14], _sqlType, _factory); }
+  { if (init) _struct = new MutableStruct(new Object[13], _sqlType, _factory); }
```

Comment:
This change is actually missing in the PR for MAAT, can we fix this before merging please?